configs/iterated-training/alternate-match-1gpu.cfg

```diff
@@ -0,0 +1,5 @@
+# Evaluation config used on every other iteration of iterated training.
+@include ../match-1gpu.cfg
+
+searchAlgorithm0 = AMCTS
+searchAlgorithm1 = MCTS
```

Comment:
Like above, maybe add a comment here denoting which one is the cyclic policy and which one is KataGo, and which one is being trained and which is frozen?